instrumentation/opentelemetry-instrumentation-aiopg/setup.cfg

```diff
@@ -39,14 +39,14 @@ package_dir=
 packages=find_namespace:
 install_requires =
     opentelemetry-api ~= 1.3
-    opentelemetry-semantic-conventions == 0.24b0
```

Comment:
why is this not required at install but is required for the test?

Reply:
Because it is not used in any code in `opentelemetry-instrumentation-aiopg/src` but it is used in `opentelemetry-instrumentation-aiopg/tests/test_aiopg_integration.py`.

Reply:
Ah i see... it relies on dbapi's use of the semantic conventions. 👍